forms.red

```diff
@@ -1,111 +1,152 @@
+staticEditorDefSize: as-pair StaticEditorDefXsize EditorDefYsize
+StaticDefOrigin: as-pair 0 (FormSheetDefYorigin + FormSheetDefYsize + 5)
+DynamicDefOrigin: as-pair StaticEditorDefXsize (FormSheetDefYorigin + FormSheetDefYsize + 5)
 DynamicCode: does copy [" "]
 
-; Database default values
-DbDefOrigin: as-pair (WindowDefXsize - (ToolboxDefXsize + 10)) FormSheetDefYorigin
-DbDefXsize: FormSheetDefXsize
-DbDefYsize: ToolboxDefYsize - 120
-DbDefSize: as-pair DbDefXsize DbDefYsize
+FontSel: attempt [make font! [name: "Arial" size: 14 style: "normal" color:FormSheetWidgetForeground ] ]
+FontDefName: "Arial"
+FontDefStyl: "Normal"
+FontDefSize: "14"
+
+forg: func[
+    clr [tuple!]
+][
+    FormSheetWidgetForeground: clr
+    unview
+    WidgetGroupFgn/color: clr
+]
+
+bacg: func[
+    clr [tuple!]
+][
+    FormSheetWidgetBackground: clr
+    unview
+    WidgetGroupBgn/color: clr
+]
+
+specified-color-f: function [][
 
-; Random color while wait for native requestor
-request-color: function [][random/seed now/time return random/secure 255.255.255.0 ]
+	to-color: function [r g b][0
+        color: 0.0.0
+        if r [color/1: to integer! 256 * r]
+        if g [color/2: to integer! 256 * g]
+        if b [color/3: to integer! 256 * b]
+        color
+    ]
+
+    to-text: function [val][form to integer! 0.5 + 255 * any [val 0]]
+
+    view [
+        title "Color sliders"
+        style txt: text 40 right
+        style value: text "0" 30 right bold
+	
+        across
+        txt "Red:"   R: slider 256 value react [face/text: to-text R/data] return
+        txt "Green:" G: slider 256 value react [face/text: to-text G/data] return
+        txt "Blue:"  B: slider 256 value react [face/text: to-text B/data]
+	
+        pad 0x-65 box: base react [face/color: to-color R/data G/data B/data] return
+
+        pad 0x20 text "The new color"
+            font  [size: 14]
+            react [face/font/color: box/color]
+        button "Done!" [do forg box/color]
+    ]
+]
+
+specified-color-b: function [][
+
+	to-color: function [r g b][0
+        color: 0.0.0
+        if r [color/1: to integer! 256 * r]
+        if g [color/2: to integer! 256 * g]
+        if b [color/3: to integer! 256 * b]
+        color
+    ]
+
+    to-text: function [val][form to integer! 0.5 + 255 * any [val 0]]
+
+    view [
+        title "Color sliders"
+        style txt: text 40 right
+        style value: text "0" 30 right bold
+	
+        across
+        txt "Red:"   R: slider 256 value react [face/text: to-text R/data] return
+        txt "Green:" G: slider 256 value react [face/text: to-text G/data] return
+        txt "Blue:"  B: slider 256 value react [face/text: to-text B/data]
+	
+        pad 0x-65 box: base react [face/color: to-color R/data G/data B/data] return
+
+        pad 0x20 text "The new color"
+            font  [size: 14]
+            react [face/font/color: box/color]
+        button "Done!" [do bacg box/color]
+    ]
+]

```

Comment:
Good job!
Before the random color, it was a full enabled color selector func from @honyx but along the time view was enhanced and the func become faulty, so I changed it by random as a time saving option 😂

Reply:
So I like your selector!

Reply:
thanks!
I thought to add your random color selector as a background color for the main window.
What do you say?